docs/fast_data/configuration/config_maps/singleViewKey.md

```diff
@@ -0,0 +1,119 @@
+---
+id: singleViewKey
+title: Single View Key Configuration
+sidebar_label: Single View Key
+---
+
+The Single View Key is a configuration file used by the Single View Creator part to identify the Single View document that needs to be updated (or if a new document must be created) as a consequence of an event that the service has consumed. 
+
+## Overview 
+
+The Single View Creator updates Single View documents every time a [Projection Changes document](/fast_data/configuration/realtime_updater.md) is received via Kafka or found in the MongoDB protection changes collection, depending on the service configuration. The content of the `identifier` property includes the list of fields and their values to be matched to the Single View document found.
+
+In case the following Projection Changes document has been received:
+```json
+{
+    "type": "sv_customer",
+    "identifier": {
+        "ID_USER": "123",
+    },
+    "changes": [{
+        "state": "NEW",
+        "topic": "my-topic.development.my-system.my-projection.ingestion",
+        "partition": 0,
+        "timestamp": "2021-11-19T16:22:07.031Z",
+        "offset": "14",
+        "key": {
+            "ID_USER": "123",
+        },
+    }],
+}
+```
+The Service understands that a projection with the identifier `ID_USER` with the value `123` must be updated by performing an aggregation because one or more values inside the Single View have been updated. However the Single View might have fields with different names, therefore a configuration to understand which field of the Single View should be compared with the `123` value to understand the document to update. This configuration is the Config Map named _Single View Key_.
+
+The Single View Key can be a JavaScript file, which must be named `singleViewKey.js` that will perform customized logic to retrieve the document to update, or a JSON file named `singleViewKey.json` with a predefined structure to simplify the relationship between the fields of the Projection Changes and the Single View.
+
+:::info
+When a Single View Creator is created from Mia Marketplace will automatically include a `singleViewKey.json` file. If you prefer to use a JavaScript function, that file should be deleted and a `singleViewKey.js` file must be created in its place.
```

Comment:
```suggestion
When a Single View Creator is created from Mia Marketplace it will automatically include a `singleViewKey.json` file. If you prefer to use a JavaScript function, that file should be deleted and a `singleViewKey.js` file must be created in its place.
```